CodeGuru/src/main/java/GuessingGame.java

```diff
@@ -0,0 +1,47 @@
+import java.util.Scanner;
+
+public class GuessingGame {
+    public static void main(String[] args) {
+        int computerNumber = (int) (Math.random()*100 + 1);
+        System.out.println("A random number has been generated...");
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter a guess between 1 and 100");
+        int userAnswer = scanner.nextInt();
+
+        // Guess 1
+        validateAnswer(userAnswer, computerNumber);
+
+        // Guess 2
+        System.out.println("Enter a guess between 1 and 100");
+        userAnswer = scanner.nextInt();
+		validateAnswer(userAnswer, computerNumber);
+
+        // Guess 3
+        System.out.println("Enter a guess between 1 and 100");
+        userAnswer = scanner.nextInt();
+		validateAnswer(userAnswer, computerNumber);
+        
+
+        System.out.println("Sorry for your luck, restart the program to play again!");
+    }
+	
+	private static void validateAnswer(userAnswer, computerNumber)
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Syntax error on token &#34;,&#34;, delete this token